docs/src/markdown-examples.md

```diff
@@ -137,9 +137,9 @@ Note the combination with `[!code focus]`.
 ```julia
 using Gnuplot
 x = -2π:0.001:2π
-@gp x sin.(x) "w l t 'sin' lw 2 lc '#56B4E9'" "set gridss" # ‎ [!code error]
+@gp x sin.(x) "w l t 'sin' lw 2 lc '#56B4E9'" "set gridss" # [!code error]
 @gp :- xrange = (-2π - 0.3, 2π + 0.3) yrange = (-1.1,1.1)
-@gp :- x cos.(x) "w l t 'cos' lw 2 lc rgba '#E69F00'" # ‎[!code warning]
+@gp :- x cos.(x) "w l t 'cos' lw 2 lc rgba '#E69F00'" # [!code warning]
```

Comment:
also, does this works for you locally? I kinda remember adding those invisible things for a reason.

Reply:
what's is the effect? maybe nowadays they are not needed, I don't remember why 😢 I used them. Well... I will merge this now.